puppeteer-tests/package.json

```diff
@@ -7,24 +7,33 @@
     "screenshot-test": "ts-node --files src/screenshot-test.ts",
     "system-test": "ts-node --files src/system-test.ts",
     "preinstall": "npx only-allow pnpm",
+    "comments-test": "jest --config ./jest.config.js",
     "build": "tsc"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@types/uuid": "8.3.0",
     "@types/node": "17.0.21",
+    "@types/uuid": "8.3.0",
+    "JSONStream": "1.3.5",
     "aws-sdk": "2.810.0",
+    "babel-jest": "^29.7.0",
```

Comment:
Please can you remove all uses of `^` in these newly added dependencies

Reply:
https://github.com/concrete-utopia/utopia/pull/4653/commits/e4cf3ba88862116de1792e38c217b2c094634527